src/lib/omerlo/reader/utils/request.ts

```diff
@@ -44,7 +44,7 @@ export async function dirtyRequest(
   }
 
   const headers = { 'Content-Type': 'application/json' }
-  const resp = await f(path.toString(), { method: opts.method, body: JSON.stringify(opts.body), headers });
+  const resp = await f(path.toString(), { method: opts?.method ?? 'get', body: JSON.stringify(opts?.body), headers });
```

Comment:
For better readability

```suggestion
  const method = opts?.method;
  const body = JSON.stringify(opts?.body);
  
  const resp = await f(path.toString(), { method, body, headers });
```